src/System.Text.Json/tests/JsonDocumentTests.cs

```diff
@@ -411,6 +411,71 @@ public static void ParseJson_UnseekableStream_Async_WithBOM(bool compactData, Te
                     GetAwaiter().GetResult());
         }
 
+        [Fact]
+        public static void ParseJson_Stream_Throws_ClearRentedBuffer_CodeCoverage()
+        {
+            byte[] data = Encoding.UTF8.GetBytes(SR.Json400KB);
+            Stream stream = new ThrowOnReadStream(data);
+            JsonDocument doc = null;
+            Assert.Throws<NotImplementedException>(() =>
+            {
+                doc = JsonDocument.Parse(stream);
+            });
+           Assert.Null(doc);
+        }
+
+        [Fact]
+        public static void ParseJson_Stream_Async_Throws_ClearRentedBuffer_CodeCoverage()
+        {
+            byte[] data = Encoding.UTF8.GetBytes(SR.Json400KB);
+            Stream stream = new ThrowOnReadStream(data);
+            JsonDocument doc = null;
+            Assert.ThrowsAsync<NotImplementedException>(async () =>
+            {
+                doc = await JsonDocument.ParseAsync(stream);
+            });
+            Assert.Null(doc);
+        }
+
+        [Fact]
+        public static void ParseJson_Stream_CanSeekThrows_NullBuffer_CodeCoverage()
+        {
+            byte[] data = Encoding.UTF8.GetBytes(SR.Json400KB);
+            Stream stream = new ThrowOnSeekStream(data);
+            JsonDocument doc = null;
+            Assert.Throws<NotImplementedException>(() =>
+            {
+                doc = JsonDocument.Parse(stream);
```

Comment:
nit: we don't need to assign the result to a variable. By definition if it throws it can't return anything :)

Reply:
I agree.

This test would be better as

```C#
[Fact]
public static void ParseJson_Stream_CanSeekThrows_NullBuffer_CodeCoverage()
{
    using (Stream stream = new ThrowOnSeekStream(new byte[] { 1 } ))
    {
        Assert.Throws<NotImplementedException>(() => JsonDocument.Parse(stream));
    }
}
```

Assuming that a payload of `1` (which is legal JSON) is enough to hit the tested condition.  (I didn't sub out the exception type here, since that's a comment from farther away)

Reply:
Yes, i know it will not return anything and @bartonjs version is enough, I wanted to show that the test is just for code coverage and nothign would happen